File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/02.lettersCombinations.js
function lettersCombinations(input) {

    let startLetter = input[0];
    let endtLetter = input[1];
    let specialLetter = input[2];
    let start = startLetter.charCodeAt();
    let end = endtLetter.charCodeAt();
    let special = specialLetter.charCodeAt();
    let counter = 0;
    let printLine = '';

    for (let i = start; i <= end; i++) {
        for (let j = start; j <= end; j++) {
            for (let k = start; k <= end; k++) {
                if (i != special && j != special && k != special) {
                    counter++;
                    printLine += `${String.fromCharCode(i)}${String.fromCharCode(j)}${String.fromCharCode(k)} `;
                }
            }
        }
    }

    console.log(printLine + counter);
}
lettersCombinations(['a', 'c', 'z']);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/02.letters_combinations.py
start_letter = input()
end_letter = input()
special_letter = input()

start = ord(start_letter)
end = ord(end_letter)
special = ord(special_letter)

counter = 0
print_line = ''

for i in range(start, end + 1):
    for j in range(start, end + 1):
        for k in range(start, end + 1):
            if i != special and j != special and k != special:
                counter += 1
                print_line += f'{str(chr(i))}{str(chr(j))}{str(chr(k))} '

print(print_line + str(counter))
